knotx-example/knotx-example-monolith/src/main/java/com/cognifide/knotx/monolith/SampleApplication.java

```diff
@@ -33,11 +27,11 @@
   public void start() {
     Vertx vertx = Vertx.vertx();
 
-    vertx.deployVerticle(new MockRemoteRepositoryVerticle(), new DeploymentOptions().setConfig(config().getJsonObject("mockRepo")));
-    vertx.deployVerticle(new MockServiceVerticle(), new DeploymentOptions().setConfig(config().getJsonObject("mockService")));
-    vertx.deployVerticle(new RepositoryVerticle(), new DeploymentOptions().setConfig(config().getJsonObject("repository")));
-    vertx.deployVerticle(new TemplateEngineVerticle(), new DeploymentOptions().setConfig(config().getJsonObject("engine")));
-    vertx.deployVerticle(new KnotxServerVerticle(), new DeploymentOptions().setConfig(config().getJsonObject("server")));
+    vertx.deployVerticle("com.cognifide.knotx.mocks.MockRemoteRepositoryVerticle", new DeploymentOptions(config().getJsonObject("mockRepo")));
```

Comment:
Please do helper method for deployment, e.g.
```java
private void deployVerticle(Vertx vertx, String verticleName, String configName) {
   vertx.deployVerticle(verticleName, new DeploymentOptions(config().getJsonObject(configName)));
}
```
And here simply invoke that method for each verticle appropriately